src/main/java/cuchaz/enigma/utils/LangUtils.java

```diff
@@ -0,0 +1,92 @@
+package cuchaz.enigma.utils;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+import com.google.common.io.Resources;
+import com.google.gson.Gson;
+
+import cuchaz.enigma.config.Config;
+
+public class LangUtils {
```

Comment:
Maybe we can name this `I18n` `Internationalization` `L10n` `Localization` which is imo better than `LangUtils`